File: CLI/Model/ResponseMessageLine.vb
Imports DocumentFormat.OpenXml.Vml.Office

Public Class ResponseMessageLine
    Public Property Type As ResponseMessageType
    Public Property Entry As String
    Public Property Detail As String

    Public Sub New(ByVal type As ResponseMessageType, ByVal entry As String, ByVal detail As String)
        Me.Type = type
        Me.Entry = entry
        Me.Detail = detail
    End Sub
End Class

Public Enum ResponseMessageType
    Information = 0
    Warning = 1
    [Error] = 2
End Enum

File: CLIPrestashopConnector/Models/ResponseMessageLine.cs
using System;
namespace CLIPrestashopConnector.Models
{
    public class ResponseMessageLine
    {
        public ResponseMessageType Type { get; set; }
        public string Entry { get; set; }
        public string Detail { get; set; }

        public ResponseMessageLine(ResponseMessageType type,string entry,string detail)
        {
            Type = type;
            Entry = entry;
            Detail = detail;
        }

    }

    public enum ResponseMessageType
    {
        Information = 0,
        Warning = 1,
        Error = 2
    }
}

